application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -46,14 +46,23 @@ Analytics.Code.AnalyticsDashboardClass_endDate.hint=End time of the date interva
 Analytics.Code.AnalyticsDashboardClass_description.hint=Additional information to be displayed.
 Analytics.Code.AnalyticsDashboardClass_DisplayedStatistics.hint=List of statistics widgets that will be displayed in the current dashboard, in order.
 
-# Live table keys
+## Config
+Analytics.Code.ConfigurationClass.matomoTrackingCode.hint=Tracking code received from Matomo. The 'u' variable might be different from the one that is initially generated by Matomo if the server you want to track is on another machine. To fix this, go to the Matomo server integration tab, copy the Matomo URL, and replace the value of 'u' with the Matomo URL.
+Analytics.Code.ConfigurationClass.idSite.hint=The ID of the site that you want to see the statistics for. This can be taken from Matomo server integration tab.
+Analytics.Code.ConfigurationClass.requestAddress.hint=The Matomo URL. This can be taken from Matomo server integration tab.
+Analytics.Code.ConfigurationClass.authToken.hint=Token generated by Matomo to be capable of using the API to view the data. This token can be generated from Settings -> Personal -> Security -> Create New Token
```

Comment:
Seeing the page screen I noticed that these translations are not displayed in the UI. This is because some things are missing from the translation key
For configuration properties, a specific key is needed in order for the translation to be used `Analytics.Code.ConfigurationClass_authToken.hint`
Can you also add translation for the label? That will be simply `Analytics.Code.ConfigurationClass_authToken`
An example https://github.com/xwikisas/application-collabora/blob/main/application-collabora-ui/src/main/resources/Collabora/Code/Translations.xml#L39
The screenshot can also be updated. Thanks!